tests/typecheck/test/test_client.yml

```diff
@@ -9,8 +9,10 @@
       reveal_type(response.client)  # N: Revealed type is "django.test.client.Client"
       reveal_type(response.context)  # N: Revealed type is "Union[django.test.utils.ContextList, builtins.dict[builtins.str, Any]]"
       reveal_type(response.content)  # N: Revealed type is "builtins.bytes"
-      reveal_type(response.redirect_chain)  # N: Revealed type is "builtins.list[Tuple[builtins.str, builtins.int]]"
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
       response.json()
+      redirected_response = client.get('foo', follow=True)
```

Comment:
We also need a test for `follow=False`.

Reply:
@sobolevn let me move the `follow` variants in a separate test case to not to clutter the `client_methods` too much. I can also test all combinations of different methods there.